lockfile/src/csharp.rs

```diff
@@ -97,7 +97,7 @@ pub struct PackageReference {
     #[serde(alias = "@Include", default)]
     pub name: String,
 
-    #[serde(alias = "@Version", alias = "@version", default)]
+    #[serde(alias = "@Version", alias = "@version", alias = "Version", default)]
```

Comment:
Do you want to include lowercase as well?

Reply:
He includes it in the test. It works with lowercase...

Reply:
Ah missed thank, thanks

Reply:
The default is lowercase, since all of these are just aliases it goes back to the field name.